plot_graph.py

```diff
@@ -44,7 +43,11 @@ def main():
     )
 
     args = parser.parse_args()
-    config_loader = config.Config.from_file(args.data_config)
+    data_config = config.Config.from_file(args.data_config)
+    xy = data_config.get_xy("state")  # (2, N_y, N_x)
```

Comment:
This should also be updated to use the datastore right? Getting the xy should be possible from any datastore.

Reply:
Yes, good point! I will do that. It should probably move to `neural_lam/` too

Reply:
Done!